app.js

```diff
@@ -1,54 +1,76 @@
 var hours = [
-  '10am: ',
-  '11am: ',
-  '12pm: ',
-  '1pm: ',
-  '2pm: ',
-  '3pm: ',
-  '4pm: ',
-  '5pm: '
+  '10am',
+  '11am',
+  '12pm',
+  '1pm',
+  '2pm',
+  '3pm',
+  '4pm',
+  '5pm'
 ];
+var stores = [];
+var tableView;
 
-Store.prototype.totals = [];
-Store.prototype.grandTotal = 0;
+// Store.prototype.totals = [];
+// Store.prototype.grandTotal = 0;
 
 function Store (storeName, minCust, maxCust, avgCookie, id) {
   this.storeName = storeName;
```

Comment:
You're declaring in an `id` parameter in your constructor, but not using the `id` anywhere.